modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavascriptClientCodegen.java

```diff
@@ -268,6 +268,8 @@ public void processOpts() {
         }
         if (additionalProperties.containsKey(USE_ES6)) {
             setUseES6(convertPropertyToBooleanAndWriteBack(USE_ES6));
+        } else {
+            setUseES6(false);
```

Comment:
The reason why I called it there is to show the line/log which templates to use, e.g.
```
[main] INFO io.swagger.codegen.languages.JavascriptClientCodegen - Using JS ES6 templates
```
So that users do not need to guess which one is being used and much easier for users to catch any issue moving forward in case the wrong version is used.